packages/types/src/documents/global/config.ts

```diff
@@ -119,6 +119,8 @@ export type AIProvider =
   | "Custom"
   | "BudibaseAI"
 
+export type AIProviderPartial = "budibase_ai" | "OpenAI" | "AzureOpenAI"
```

Comment:
How come this is `budibase_ai` and not `BudibaseAI`?

Reply:
Fixed